packages/browser/src/index.ts

```diff
@@ -248,4 +310,20 @@ export default class LogtoClient {
       throw new LogtoClientError('invalid_id_token', error);
     }
   }
+
+  private saveCodeToken({
+    refreshToken,
+    idToken,
+    scope,
+    accessToken,
+    expiresIn,
+  }: CodeTokenResponse) {
+    this.refreshToken = refreshToken;
+    this.idToken = idToken;
+
+    // NOTE: Add scope to accessTokenKey when needed. (Linear issue LOG-1589)
```

Comment:
Looks like you have scope passed in?

Reply:
Intended to notice us add the scope in the future.

Updated to `// NOTE: Will add scope to accessTokenKey when needed. (Linear issue LOG-1589)` (add **will**) for exact meaning.